src/nodes/linkBlock.ts

```diff
@@ -0,0 +1,75 @@
+import { MyNodeSpec, NodeGroups } from './types';
+import { MdFormatSerialize, TexFormatSerialize } from '../serialize/types';
+import { StaticPhrasingContent, LinkBlock } from '../spec';
+
+export interface Attrs {
+  title: string;
+  description: string;
+  url: string;
+}
+
+const link_block: MyNodeSpec<Attrs, LinkBlock> = {
+  attrs: {
+    url: { default: '' },
+    title: { default: '' },
+    description: { default: '' },
+  },
+  group: NodeGroups.top,
+  content: `${NodeGroups.text}*`,
```

Comment:
Does this node have children from the schema perspective?
I think it might at least be an atom: true?